lib/checks/keyboard/focus.js

```diff
@@ -0,0 +1,7 @@
+if (node.getAttribute('onFocus') === 'this.blur()') {
```

Comment:
This should account for whitespace. Simply putting `.trim()` on the attribute value should do.